src/main/kotlin/graphics/scenery/ui/menu.kt

```diff
@@ -0,0 +1,40 @@
+package graphics.scenery.ui
+
+import glm_.vec2.Vec2
+import graphics.scenery.Node
+import graphics.scenery.Scene
+import imgui.dsl
+import imgui.ImGui
+import java.text.SimpleDateFormat
+
+object menu {
```

Comment:
Wrong case

Reply:
Also the name could be a bit more specific :-) Maybe that code should actually be moved into the example?